lib/gui/index.html

```diff
@@ -57,7 +57,8 @@
         isFinish: state.is('success')
       }">
       <safe-webview
-        src="'https://etcher.io/success-banner/'"></safe-webview>
+        src="'https://etcher.io/'"
+        on-state-change="state.onStateChange"></safe-webview>
```

Comment:
@jviotti It's the 'cleanest' way I've found so far, given the React/Angular mixing situation we're in. Basically, the React element isn't re-rendered when the attributes change, so we can't just pass an Angular variable and cause a re-render whenever that changes, and I suspect we won't want to do that anyway because if attribute changes caused a re-render it would refresh the webview. I'll look into if there's a better way though!

@lurch `onStateChange` is what it says on the tin, but way it's used is more like `onLeaveSuccessPage` because the first-class function we pass to `onStateChange` checks if the state change is from `'success`', so we could call the function `onStateChange` takes `onLeaveSuccessPage`.